packages/fuselage-hooks/src/useDebouncedUpdates.spec.ts

```diff
@@ -1,50 +1,38 @@
-import { createElement, useState, FunctionComponent, StrictMode } from 'react';
-import { render } from 'react-dom';
-import { act } from 'react-dom/test-utils';
+import { renderHook, act } from '@testing-library/react-hooks';
+import { useState } from 'react';
 
-import { useDebouncedUpdates } from '.';
+import { useDebouncedUpdates } from './useDebouncedUpdates';
 
-describe('useDebouncedUpdates hook', () => {
```

Comment:
We won't use `describe` anymore?

Reply:
Jest doesn't require it, as the test suite filename itself works to name the test suite -- it goes into the cli output regardless. Better keep using `describe` only to limit special conditions for tests, like sections that require `before` and `after` hooks.